### ISSUE TITLE
Footer very light color

### ISSUE THREAD

**Issue by yatil** [w3c/web-roadmaps#232]
Contrast should be at 4.5:1 

Currently: #a2b1c1 / #ffffff 2.19:1
Should be #7676767 / #ffffff 4.5:1
